models/index.js

```diff
@@ -7,17 +7,60 @@ exports.fetchCategories = () => {
   });
 };
 
-exports.fetchReviews = () => {
-  const queryStr = `
-  SELECT reviews.review_id, reviews.title, reviews.designer, reviews.owner, reviews.review_img_url, reviews.category,
-  reviews.created_at, reviews.votes, COUNT(comment_id)::int AS comment_count FROM reviews
-  JOIN comments
-  ON reviews.review_id = comments.review_id
-  GROUP BY reviews.review_id
-  ORDER BY created_at desc;`;
-  return db.query(queryStr).then(({ rows: reviews }) => {
-    return reviews;
-  });
+exports.fetchReviews = (category, sort_by = "created_at", order = "desc") => {
+  const validSortByQueries = [
+    "review_id",
+    "title",
+    "owner",
+    "category",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+  const validOrdersQueries = ["asc", "desc"];
+  const validCategoriesQueries = [
```

Comment:
Because this is row information we shouldn't green list them, its possible that we create an end point to add a category, plus you only have the test categories in here, this can and should be done without greenlisting